pkg/utils/utils.go

```diff
@@ -280,19 +280,19 @@ func MonitorMounts() {
 					continue
 				}
 				if _, ok := TransitionVolList[vol.Spec.Volume.Name]; !ok {
```

Comment:
Can we add check if the deletion timestamp is set we shouldn't monitor for such resources...